library/zlib/deflate/deflate_spec.rb

```diff
@@ -1,22 +1,24 @@
-# -*- encoding: us-ascii -*-
```

Comment:
@eregon parser reads the magic encoding comment and then converts from that encoding to utf-8 just fine. It is later (in the lexer I suppose) where invalid strings are rejected "escape sequences incompatible with UTF-8"